src/errorpage.cc

```diff
@@ -758,6 +758,22 @@ ErrorState::Convert(char token, bool building_deny_info_url, bool allowRecursion
             p = "-";
         break;
 
+    case 'A': {
+        char *tmp = NULL;
+        const bool interceptedAtKnownPort = request ?
+                                            (request->flags.interceptTproxy ||
+                                             request->flags.intercepted) &&
+                                            request->masterXaction->squidPort :
+                                            false;
+        if (interceptedAtKnownPort) {
+            if (!request->masterXaction->squidPort->s.isAnyAddr())
+                tmp = request->masterXaction->squidPort->s.toStr(ntoabuf, MAX_IPSTRLEN);
+        } else if (request->masterXaction->tcpClient)
+            tmp = request->masterXaction->tcpClient->local.toStr(ntoabuf, MAX_IPSTRLEN);
+        mb.appendf("%s", tmp ? tmp : getMyHostname());
```

Comment:
We should not give a domain name after promising an IP address.

Reply:
What do you suggest we should use if information is not available? I chose hostname because that MAY then resolve to listening IP. 

PS: On vacation for 4-5 days. So requested changes may take time.

Reply:
> What do you suggest we should use if information is not available?

I suggest using a single dash (`-`) but do not insist on it. The current situation is messy:
* most of the error page assembly code seems to be using a `[no ...]` pattern
* some of the error page assembly code uses a dash
* future logformat-driven error page assembly code will use a dash (for logformat %codes)

AFAICT, the missing information placeholder should be a constant string that is easy to identify in scripts and such. A domain name does not satisfy that requirement (one could add another %code to output the domain name if needed, of course).

`deny_info` may place its own special requirements on the placeholder -- I have not checked, but you should.